macros/edr/tests/test_utils/get_anomaly_query.sql

```diff
@@ -51,15 +51,20 @@
       anomaly_scores_with_is_anomalous as (
         select
           *,
-          case when
+case when
             anomaly_score is not null and
-                          {{ elementary.is_score_anomalous_condition(test_configuration.anomaly_sensitivity, test_configuration.anomaly_direction) }} and
-                          {{ elementary.avg_percent_anomalous_condition(
-                            test_configuration.ignore_small_changes.spike_failure_percent_threshold,
-                            test_configuration.ignore_small_changes.drop_failure_percent_threshold,
-                            test_configuration.anomaly_direction
-                          ) }} and
-                          bucket_end >= {{ elementary.edr_timeadd('day', backfill_period, 'max_bucket_end') }} and
+            (
+              (
```

Comment:
maybe you should reorder it?
the fail on zero calculation should be easier, could have better performace